app/models/manageiq/providers/vmware/cloud_manager/refresher.rb

```diff
@@ -13,4 +13,37 @@ def save_inventory(ems, target, hashes)
   def post_process_refresh_classes
     [::Vm]
   end
+
+  def collect_inventory_for_targets(ems, targets)
```

Comment:
you should be able to get these by inheriting from ManagerRefresher

see e.g. https://github.com/Ladas/manageiq-providers-amazon/blob/22c9d9614067361b26fc9ca6ad7f028512b9de9a/app/models/manageiq/providers/amazon/cloud_manager/refresher.rb#L1